Symfony/CS/Linter/Linter.php

```diff
@@ -106,7 +106,8 @@ private function createProcessForFile($path)
             return $this->createProcessForSource(file_get_contents($path));
         }
 
-        $process = new Process(sprintf('%s -l %s', $this->executable, ProcessUtils::escapeArgument($path)));
+        $executable = ProcessUtils::escapeArgument($this->executable).(defined('HHVM_VESION') ? '--php' : '');
```

Comment:
> Executable as it is may not be escaped. When you find executable you may not want it escaped, it's up to your use case.

I don't really agree. If we're including the "arguments" appended to it, then it should definitely be quoted. I think that's a bug, but changing it would be a massive break. I'm not sure what symfony will actually do here.